source/backups.rst

```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
               backupName: backup1
             EOF
 
+.. _backups-pitr-restore:
+
+Restoring backup with point-in-time recovery
+********************************************
+
+Following steps are needed to to roll back the cluster to a
+specific date and time:
+
+Details:
+
+1. First of all make sure that the cluster is running.
+
+2. Now find out correct name for the **cluster** (backup name is not needed if
+   point-in-time recovery is enabled and will be ignored). Available
+   clusters can be listed with the following command:
```

Comment:
This is not right. In the spec we say that backupName is needed. We do that to ensure that user knows which backup to use and where oplogs are. It is to cover the case where user adds new S3 bucket if PITR is enabled and some other weird  cases.